code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -664,3 +664,111 @@
+	for(var/i in 1 to 2)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/soybeans(src)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/chili(src)
+
+/obj/item/weapon/storage/box/ingredients/italian
+	item_state = "italian"
+
+/obj/item/weapon/storage/box/ingredients/italian/New()
+	..()
+	for(var/i in 1 to 3)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/tomato(src)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/vulgaris(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/faggot(src)
+
+/obj/item/weapon/storage/box/ingredients/vegetarian
+	item_state = "vegetarian"
+
+/obj/item/weapon/storage/box/ingredients/vegetarian/New()
+	..()
+	for(var/i in 1 to 3)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/vulgaris(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/carrot(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/eggplant(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/potato(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/apple(src)
+
+/obj/item/weapon/storage/box/ingredients/sweets
+	item_state = "sweets"
+
+/obj/item/weapon/storage/box/ingredients/sweets/New()
+	..()
+	for(var/i in 1 to 3)
+		new /obj/item/weapon/reagent_containers/food/snacks/chocolatebar(src)
+	new /obj/item/weapon/reagent_containers/food/condiment/sugar(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/cherries(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/grown/banana(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/icecream(src)
+
+/obj/item/weapon/storage/box/ingredients/carnivore
+	item_state = "carnivore"
+
+/obj/item/weapon/storage/box/ingredients/carnivore/New()
+	..()
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/bear(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/spider(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/carpmeat(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/slime(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/faggot(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/corgi(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/monkey(src)
+
+/obj/item/weapon/storage/box/ingredients/exotic
+	item_state = "exotic"
+
+/obj/item/weapon/storage/box/ingredients/exotic/New()
+	..()
+	new /obj/item/weapon/reagent_containers/food/condiment/soysauce(src)
+	for(var/i in 1 to 2)
+		new /obj/item/weapon/reagent_containers/food/snacks/grown/cabbage(src)
+		new	/obj/item/weapon/reagent_containers/food/snacks/soydope(src)
+		new /obj/item/weapon/reagent_containers/food/snacks/carpmeat(src)
+
+/obj/item/weapon/storage/box/ingredients/New()
+	..()
+	if (item_state != null)
+		desc = "A box containing supplementary ingredients for the aspiring chef. This box's theme is '[item_state]'."
```

Comment:
Not sure this bit is really necessary, could just add a desc to each box normally (unless you want them to all follow this format I guess)